### DISCUSSION TITLE
Alphafold - DLBinder

### DISCUSSION THREAD

**Discussion by zarbuk** [RosettaCommons/RFdiffusion#223] [Q&A] [answered]
Hello everyone, so I have been able to run RF diffusion to get some backbones and PRotein MPNN for the sequences however, I am facing issues with Alphafold installation , I do use GPU but we dont have NVIDIA microchip GPU instaed what we have is 08:00.0 VGA compatible controller: ASPEED Technology, Inc. ASPEED Graphics Family (rev 30)
Can anyone help me either edit the yml file or if there are some other ways that I can run and install Alphafold on cluster  specifically for DLBinder design?

**Comment by roccomoretti** [comment, marked as answer]
If you're having trouble installing AlphaFold, I'd recommend asking on an AlphaFold-specific forum. I will note that you don't need to install AlphaFold in the same Conda environment you run RFDiffusion in. You can always do a `conda activate` command between running RFDiffusion and running AlphaFold. Using a different Conda environment might make things easier, as you don't have to coordinate dependencies.

I'll also note that there's nothing particularly special about AlphaFold in the RFDiffusion/ProteinMPNN/AlphaFold pipeline. The benefits AlphaFold has is that it's (by many metrics) the best performing sequence->structure method at the moment and that it's orthogonal to the RoseTTAFold model that underlies RFDiffusion (so less chance you'll get something RoseTTAFold anomalously likes). But theoretically any sequence->structure method should be usable: AlphaFold, OmegaFold, OpenFold, ColabFold, RoseTTAFold, ESMFold, what have you. Feel free to use whatever is easiest in your hands. The choice of method might affect your success rate, but it's only those failures which inaccurately predict your design to fold to the same structure that RFDiffusion generated which will affect your wet-lab success rate. (Designs which are inaccurately predicted to misfold only waste computer time.)

**Reply by zarbuk** [reply]
Thank you
